lib/telemetry.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+import { InfluxDB } from '@influxdata/influxdb-client'
+import { createHash } from 'node:crypto'
+import Sentry from '@sentry/node'
+import assert from 'node:assert'
+
+const { FIL_WALLET_ADDRESS } = process.env
+
+const client = new InfluxDB({
+  url: 'https://eu-central-1-1.aws.cloud2.influxdata.com',
+  token:
+    // station-anonymous-write
+    '0fZyu9zjDvYlaNfOeuwgnQoUI0VcSzeYDpnOLjQyr30mz-Plqels5JHEwgKRbtCcDJbQmv62VnOV_FsZVxgoow=='
+})
+
+export const writeClient = client.getWriteApi(
+  'Filecoin Station', // org
+  'station', // bucket
+  'ns' // precision
+)
+
+setInterval(() => {
+  writeClient.flush().catch(err => {
+    // Ignore unactionable InfluxDB errors
+    if (
+      /HttpError|getAddrInfo|RequestTimedOutError|ECONNRESET/i.test(String(err))
+    ) {
+      return
+    }
+    Sentry.captureException(err)
+  })
+}, 5000).unref()
+
+export const writePoint = point => {
+  assert(FIL_WALLET_ADDRESS)
+  point.stringField(
+    'wallet',
+    createHash('sha256').update(FIL_WALLET_ADDRESS).digest('hex')
+  )
+  point.tag('station', 'core')
```

Comment:
This way we can tell Desktop and Core stats apart

Reply:
We can add the tag to `desktop`, but there's no pressure since no tag also means `desktop`.

And good point about overwriting the env 👍